lib/Elastica/AbstractScript.php

```diff
@@ -1,27 +1,10 @@
 <?php
 namespace Elastica;
 
+use Elastica\Script\AbstractScript as BaseAbstractScript;
 /**
- * Base class for Script object.
- *
- * @author Nicolas Assing <nicolas.assing@gmail.com>
- *
- * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting.html
+ * Kept for BC reasons.
```

Comment:
Can we add a @deprecated message here? Check some other files which are deprecated for some examples.